### PR TITLE
Feature/fix jel 43

### PR DESCRIPTION
**Pull request by mafernando**
This PR fixes [JEL-45](https://boozallen.atlassian.net/browse/JEL-43) by addressing issues with project.monthly_spend being broken in RSpec.

### CODE REVIEW

File: app/use_cases/create_service_order.rb [hunk 1]
```diff
@@ -63,7 +61,7 @@ def build_order
       hourly_price: product.hourly_price,
       monthly_price: product.monthly_price,
       service: service
-    )
+    ).except(:name)
```
Comment (stackus) [inline app/use_cases/create_service_order.rb]: Why did you remove the services bit above only to exclude it here.

This needs to be undone because UX is going to be sending in a services object with a name value.

Comment (mafernando) [inline app/use_cases/create_service_order.rb]: Should be addressed in PR https://github.com/projectjellyfish/api/pull/908
